src/frontend/js/modules/esn.md-menu.backdrop.js

```diff
@@ -0,0 +1,25 @@
+angular.module('esn.md-menu.backdrop', [])
+  .directive('mdMenuBackdropFix', function() {
```

Comment:
the directive name seems like a :adhesive_bandage: name

Reply:
Because it's exactly what it means.